src/client/NuGet.Client.Staging/StagingClient.cs

```diff
@@ -67,7 +65,7 @@ public StagingClient(Uri stagingServiceUri, ITestOutputHelper logger)
 
         public async Task<JObject> GetDetails(string stageId)
         {
-            _logger.WriteLine($"StagingClient: GetDetails called for stage {stageId}");
+            _logger.LogDebug($"StagingClient: GetDetails called for stage {stageId}");

```

Comment:
I can't assume that the user of this code is using serilog, so I would prefer to use regular logs

---
In reply to: [66213997](https://github.com/NuGet/NuGet.Staging/pull/62#discussion_r66213997) [](ancestors = 66213997)